### PR TITLE
Adding support for number of turns for TERNCY-SD01

### PR DESCRIPTION
**Pull request by JuliusSenkus**
Adding missing property to get how much rotary switch was turned.

### CODE REVIEW

File: devices/terncy.js [hunk 1]
```diff
@@ -38,7 +38,7 @@ module.exports = [
         ota: ota.zigbeeOTA,
         meta: {battery: {dontDividePercentage: true}},
         exposes: [e.battery(), e.action(['single', 'double', 'triple', 'quadruple', 'rotate']),
-            exposes.text('direction', ea.STATE)],
+            exposes.text('direction', ea.STATE), exposes.numeric('number', ea.STATE).withDescription('Number of turns')],
```
Comment (Koenkk) [inline devices/terncy.js:41]: Shouldn't `number` be `action_number`?

Comment (JuliusSenkus) [reply devices/terncy.js:41]: No, if it should, also instead `direction` there should be `action_direction`. Also tested it.
image

Comment (JuliusSenkus) [reply devices/terncy.js:41]: Or am I unaware of some convention?

Comment (Koenkk) [reply devices/terncy.js:41]: Ah sorry, you still have the legacy integration enabled, try disabling it: https://www.zigbee2mqtt.io/devices/TERNCY-SD01.html#device-type-specific-configuration and make sure to remove your `data/state.json` to purge the old attributes.

Comment (JuliusSenkus) [reply devices/terncy.js:41]: Stopped service, set `legacy: false` for this device in `devices.yaml`, removed `state.json` and that seemed to break my scripts for this device, did not investigate much (maybe missing states), why was that needed?

**Pull request closed without merging**